### PR TITLE
Fix integration tests after bumping deps

### PR DESCRIPTION
**Pull request by noisersup**
# Description

Removed the test as it tests driver and not the database.

## Readiness checklist

- [ ] I added/updated unit tests.
- [x] I added/updated integration/compatibility tests.
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: integration/basic_test.go [hunk 1]
```diff
@@ -524,7 +524,7 @@ func TestDatabaseName(t *testing.T) {
 		ctx, collection := setup.Setup(t)
 
 		err := collection.Database().Client().Database("").CreateCollection(ctx, collection.Name())
-		expectedErr := driver.InvalidOperationError(driver.InvalidOperationError{MissingField: "Database"})
+		expectedErr := errors.New("database name cannot be empty")
```
Comment (AlekSi) [inline integration/basic_test.go:527]: That error is returned by the driver, not FerretDB or MongoDB. We should just delete that test.

Comment (noisersup) [reply integration/basic_test.go:527]: That's right

Comment (AlekSi) [reply integration/basic_test.go]: We could, in theory, write a test that uses `RunCommand` to do `insert`… But let's not do that now.

**Review by AlekSi — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
Thanks for the quick fix! It answers the exact question I was about to ask :)

**Pull request merged**